services/nvidia-gpu-operator/25.3.0/defaults/cm.yaml

```diff
@@ -43,3 +43,8 @@ data:
     operator:
       repository: nvcr.io/nvidia
       version: v25.3.0
+    migManager:
```

Comment:
Because this is an operator our tooling cannot discover some images only by static analysis. For such cases we maintain list of additional images that are included in the bundle.

Can you add additional images to this list https://github.com/mesosphere/kommander-applications/blob/main/services/nvidia-gpu-operator/25.3.0/helmrelease/extra-images.txt